package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-lds",
-  "version": "7.3.2",
+  "version": "7.3.3",
```

Comment:
an inconsistency of versions in `package.json` and `CHANGELOG.md`

Reply:
fixed already